examples/rawkv/rawkv.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
Code should belong to tikv (aka CNCF) instead.

Reply:
Copyright 2021 TiKV Authors